cmd/platform/user.go

```diff
@@ -202,7 +201,7 @@ func userDeactivateCmdF(cmd *cobra.Command, args []string) error {
 	}
 
 	if len(args) < 1 {
-		return errors.New("Enter user(s) to deactivate.")
+		return errors.New("Expected at least one argument. See help text for details.")
```

Comment:
Which help text? Wondering if we can include which argument is expected, i.e. a user.

Similarly for the others